.gitmodules

```diff
@@ -41,3 +41,9 @@
 [submodule "meta-browser"]
 	path = meta-browser
 	url = git://github.com/OSSystems/meta-browser.git
+[submodule "renesas-rcar-gen3"]
+	path = renesas-rcar-gen3
+	url = https://github.com/slawr/renesas-rcar-gen3.git
```

Comment:
@slawr No but it would be a sad-faced merge. :) I don't think more details should be needed from GDP team. It all depends on how Renesas devs decide to solve the issue.